src/server.js

```diff
@@ -56,6 +57,7 @@ export default class Tapestry {
     this.server.start(err => {
       if (err) error(err)
       success(`Server ready: ${this.server.info.uri}`)
+      if (typeof this.cb === 'function') this.cb()
```

Comment:
If `new Tapestry` returns a server object, we should have access to all of Hapi's server events: https://hapijs.com/api/8.8.1#server-events

```js
const server = new Tapestry(config)

server.on('start', new Build({cwd, config}))
server.on('stop', exec('cowsay "Server stopped!"'))
```

And then we could do something lik